bin/src/tx.rs

```diff
@@ -117,8 +116,8 @@ impl TxCmd {
                 updates,
                 app_updates,
             } => {
-                let updates = from_json_str(&updates)?;
-                let app_updates = from_json_str(&app_updates)?;
+                let updates = <_>::from_json_str(&updates)?;
+                let app_updates = <_>::from_json_str(&app_updates)?;
```

Comment:
A drawback of extension trait is we need to explicitly write out the type `T` when calling functions like `T::from_json_str`.